addOns/automation/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Fixed
 - Bug in handling headers with colons in the values.
+- When loading Authentication Verification details don't override the authentication method's polling frequency with zero if the authentication method already has a value.
```

Comment:
I don't really like this, but I need a suggestion. I re-wrote it a few times and still can't land on something I feel is good.